lisa_commands/lambda_function.py

```diff
@@ -35,6 +35,8 @@ def lambda_handler(data, _context):
         }
         if 'thread_ts' in slack_event:
             data['thread_ts'] = slack_event['thread_ts']
+        if 'unfurl_media' in slack_event:
```

Comment:
This line brings up a question.

Do we want the bot to unfurl media or not. I feel like we should make it an all or nothing
thing. Because ticking it on and off for various commands seems like unneeded complexity.

Actually I think I'll poll the slack chat tomorrow and see what people think. 

Sound good?

Reply:
Yep that's good with me